AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/CMakeLists.txt

```diff
@@ -22,17 +22,6 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
             AZ::AssetProcessor
     )
 
-    ly_add_pytest(
-        NAME AssetPipelineTests.Gui_2_Main
-        PATH  ${CMAKE_CURRENT_LIST_DIR}/asset_processor_gui_tests_2.py     
-        EXCLUDE_TEST_RUN_TARGET_FROM_IDE
-        TEST_SUITE main
-        PYTEST_MARKS "not SUITE_sandbox" # don't run sandbox tests in this file
```

Comment:
is there a reason you're pulling the whole module out here, instead of just marking the tests that have issues with the SUITE_sandbox flag?

Reply:
With the exception of test_WindowsAndMacPlatforms_GUIFastScanNoSettingSet_FastScanSettingCreated, and the two tests being disabled here, all other tests in the module are currently sandboxed.

I couldn't find anything that would make the tests fail/frail on my end.
One is a copy job with a text file and the other is processing a blank prefab